### ISSUE TITLE
robust 2x2 extensions not cleaned with setup.py clean rule

### ISSUE THREAD

**Issue by bennahugo** [ratt-ru/CubiCal#249]
```
 - 09:02:44 - data_handler       [5.4/6.4 11.4/11.4 2.5Gb]   computing visibilities for input/pks-1934-63-only.txt
2019-02-07 09:02:55.813752: I tensorflow/core/platform/cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled
 to use: SSE4.1 SSE4.2 AVX AVX2 FMA
 - 09:03:37 - solver             [1.9/10.1 25.3/33.5 3.9Gb] Solver for tile 0 chunk D0T0F0 failed with exception: 'module' object has no attribute 'allocate
_vis_array'
 - 09:03:37 - solver             [1.9/10.1 25.3/33.5 3.9Gb] Traceback (most recent call last):
  File "/scratch/bhugo/CubiCal/cubical/solver.py", line 764, in run_solver
    allocator=kernel.allocate_vis_array,
AttributeError: 'module' object has no attribute 'allocate_vis_array'
 - 09:03:37 - main               [1.9/10.1 25.3/33.5 3.9Gb] Exiting with exception: AttributeError('module' object has no attribute 'allocate_vis_array')
 Traceback (most recent call last):
  File "/scratch/bhugo/CubiCal/cubical/main.py", line 363, in main
    stats_dict = workers.run_process_loop(ms, tile_list, load_model, single_chunk, solver_type, solver_opts, debug_opts)
  File "/scratch/bhugo/CubiCal/cubical/workers.py", line 210, in run_process_loop
    return _run_single_process_loop(ms, load_model, single_chunk, solver_type, solver_opts, debug_opts)
  File "/scratch/bhugo/CubiCal/cubical/workers.py", line 325, in _run_single_process_loop
    solver.run_solver(solver_type, itile, key, solver_opts, debug_opts)
  File "/scratch/bhugo/CubiCal/cubical/solver.py", line 764, in run_solver
    allocator=kernel.allocate_vis_array,
AttributeError: 'module' object has no attribute 'allocate_vis_array'
```

**Comment by ulricharmel** [comment]
Hi @bennahugo I am not sure why you are getting this issue. Do you have the latest version of the Robust Solver? I don't have this issue on  my installation.  The 'allocate_vis_array' here https://github.com/ratt-ru/CubiCal/blob/master/cubical/kernels/includes/complex_w_gain_kernel.pxi#L19 in the file **complex_w_gain_kernel.pxi** which is included in the specifed kernel here https://github.com/ratt-ru/CubiCal/blob/master/cubical/kernels/cyfull_W_complex.pyx#L144 and that is the kernel which get_kernel returns https://github.com/ratt-ru/CubiCal/blob/master/cubical/machines/complex_W_2x2_machine.py#L69

**Comment by bennahugo** [comment]
Hmm... I've repulled master and generated the cython kernels yesterday. I'll start a clean installation and recheck towards EOB today.

**Comment by bennahugo** [comment]
@ulricharmel never mind.. it looks like it is a bug somewhere in the extension building and cleaning logic - pulling cubical from scratch works, but running setup.py clean setup.py gocythonize and setup.py build left it out of date. I'll let you know how it goes, not getting fantastic results with the normal solver on a calibrator field

**Comment by JSKenyon** [comment]
Fixed in #288.
